user/how-to-guides/how-to-use-optical-discs.md

```diff
@@ -10,7 +10,11 @@ ref: 204
 title: How to use optical discs
 ---
 
-Passthrough reading and recording (a.k.a., "burning") are not supported by Xen.
+Passthrough reading and recording (a.k.a., "burning") are not supported by Qubes OS.
+=======
```

Comment:
This will create an h1 heading, which is not what you want here, since every page already has its h1 generated from the title. (Also, this sentence just wouldn't be appropriate as an h1 heading.)